contracts/core/SetToken.sol

```diff
@@ -1,110 +1,126 @@
+/*
+    Copyright 2018 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
 pragma solidity 0.4.24;
 pragma experimental "ABIEncoderV2";
 
 
-import "zeppelin-solidity/contracts/token/ERC20/StandardToken.sol";
-import "zeppelin-solidity/contracts/token/ERC20/ERC20.sol";
-import "zeppelin-solidity/contracts/token/ERC20/DetailedERC20.sol";
-import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import { DetailedERC20 } from "zeppelin-solidity/contracts/token/ERC20/DetailedERC20.sol";
+import { ERC20 } from "zeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import { StandardToken } from "zeppelin-solidity/contracts/token/ERC20/StandardToken.sol";
+import { SafeMath } from "zeppelin-solidity/contracts/math/SafeMath.sol";
 import "../lib/AddressArrayUtils.sol";
-import "./interfaces/ISetToken.sol";
 
 
 /**
  * @title {Set}
  * @author Felix Feng
+ *
  * @dev Implementation of the basic {Set} token.
+ *
  */
 contract SetToken is
     StandardToken,
-    DetailedERC20("", "", 18),
-    ISetToken
+    DetailedERC20("", "", 18)
```

Comment:
To be careful here, we may want to be passing in the details in the constructor of the name and symbol. This means we may need to deprecate usage of DetailedERC20